client/src/components/EditProfile/ProfilePhoto/UploadPhoto.tsx

```diff
@@ -18,7 +18,7 @@ const UploadPhoto: React.FC = (): JSX.Element => {
     setSelectedImage(fileList[0]);
   };
 
-  const uploadImage = async () => {
+  const uploadImage = useCallback(async () => {
```

Comment:
Alternatively, you could have moved the definition of `uploadImage` into the `useEffect` hook to circumvent the dependency issue 👍🏼